### PR TITLE
Removed UsePersistentCookiesForExplorerView requirement

### PR DESCRIPTION
**Pull request by PramodBalusu**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/sharepoint-view-in-edge.md [hunk 1]
```diff
@@ -94,13 +94,7 @@ Follow the steps below to use View in File Explorer in Microsoft Edge:
...
         > Ensure the management shell version is 16.0.21610.12000 or higher or the ViewInFileExplorerEnabled option will not be available.
 
-7. **(Required)** Next, as a tenant administrator, update your SharePoint Online tenant configuration via SharePoint Online Management Shell to allow persisted cookies for View with Explorer.
-
-1. Run the following cmdlet to enable persistent cookies.
-
-    ```PowerShell
-    Set-SPOTenant -UsePersistentCookiesForExplorerView $true
-    ```
+7. **(Required)** "View in File Explorer" requires persisted cookies to operate correctly, user need to select "Yes" when prompted for "Stay signed in?" at the time of sign-in.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sharepoint-view-in-edge.md:97]:
```suggestion
7. **(Required)** _View in File Explorer_ requires persisted cookies to operate correctly; you need to select **Yes** when prompted to **Stay signed in?** when you sign in.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - @PramodBalusu, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@adeejoseph PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**